docs/modules/secret-operator/pages/secretclass.adoc

```diff
@@ -48,13 +70,15 @@ spec:
           name: secret-provisioner-tls-ca
           namespace: default
         autoGenerate: true
+      maxCertificateLifetime: 15d # optional
 ----
 
 `autoTls`:: Declares that the `autoTls` backend is used.
 `autoTls.ca`:: Configures the certificate authority used to issue `Pod` certificates.
 `autoTls.ca.secret`:: Reference (`name` and `namespace`) to a K8s `Secret` object where the CA certificate and key is stored in the keys `ca.crt`
                       and `ca.key` respectively.
 `autoTls.ca.autoGenerate`:: Whether the certificate authority should be provisioned if it can not be found.
+`autoTls.maxCertificateLifetime`:: Maximum lifetime the created certificates are allowed to have. In case consumers request a longer lifetime than allowed by this setting, the lifetime will be the minimum of both, so this setting takes precedence.
```

Comment:
```suggestion
`autoTls.maxCertificateLifetime`:: Maximum lifetime the created certificates are allowed to have. In case consumers request a longer lifetime than allowed by this setting, the lifetime will be the minimum of both.
```